frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTableConfig.tsx

```diff
@@ -99,178 +97,145 @@ const generateTableHeaders = (options: {
 }): IDataColumn[] => {
   const { selectedTeamId, tableType, canAddOrDeletePolicy } = options;
 
-  switch (tableType) {
-    case "inheritedPolicies":
-      return [
-        {
-          title: "Name",
-          Header: "Name",
-          disableSortBy: true,
-          accessor: "name",
-          Cell: (cellProps: ICellProps): JSX.Element => (
+  const tableHeaders: IDataColumn[] = [
+    {
+      title: "Name",
+      Header: "Name",
+      disableSortBy: true,
+      accessor: "name",
+      Cell: (cellProps: ICellProps): JSX.Element => (
+        <LinkCell
+          classes="w250-sm"
+          value={cellProps.cell.value}
+          path={PATHS.EDIT_POLICY(cellProps.row.original)}
+        />
+      ),
+    },
+    {
+      title: "Yes",
+      Header: () => <PassingColumnHeader isPassing />,
+      disableSortBy: true,
+      accessor: "passing_host_count",
+      Cell: (cellProps: ICellProps): JSX.Element => {
+        if (cellProps.row.original.has_run) {
+          return (
             <LinkCell
-              value={cellProps.cell.value}
-              path={PATHS.EDIT_POLICY(cellProps.row.original)}
-              classes="" // Override default
+              value={`${cellProps.cell.value} host${
+                cellProps.cell.value.toString() === "1" ? "" : "s"
+              }`}
+              path={createHostsByPolicyPath(
+                cellProps.row.original.id,
+                PolicyResponse.PASSING,
+                selectedTeamId
+              )}
             />
-          ),
-        },
-      ];
-    default: {
-      const tableHeaders: IDataColumn[] = [
-        {
-          title: "Name",
-          Header: "Name",
-          disableSortBy: true,
-          accessor: "name",
-          Cell: (cellProps: ICellProps): JSX.Element => (
+          );
+        }
+        return (
+          <>
+            <span
+              className="text-cell text-muted has-not-run tooltip"
+              data-tip
+              data-for={`passing_${cellProps.row.original.id.toString()}`}
+            >
+              ---
+            </span>
+            <ReactTooltip
+              place="bottom"
+              effect="solid"
+              backgroundColor="#3e4771"
```

Comment:
Should we use something from `COLORS` here?

Reply:
Rachel called this out earlier in this PR and the issue is this color is not in Mike Ts new styleguide yet I've asked him if we should change the value or update the styleguide and waiting for his decision. for now am just gonna leave it how it is.